packages/clay-css/src/scss/mixins/_popovers.scss

```diff
@@ -85,6 +88,10 @@
 			.inline-scroller {
 				@include clay-css(map-get($map, inline-scroller));
 			}
+
+			&.popover-width-wider {
+				@include clay-css(map-get($map, wider));
+			}
```

Comment:
@pat270 maybe the naming for this class could be `.popover-width-lg` what do you think? maybe it should better fit the size standards we have?

Reply:
`popover-width-lg` does fit better, we should do that.